ui/src/routes.ts

```diff
@@ -51,6 +52,10 @@ export const ROUTES: Routes = {
     layout: "sider",
     path: "/credentials/issue/:schemaID?",
   },
+  issuedQR: {
+    layout: "fullWidthGrey",
+    path: "/credentials/scan-issued/:credentialID",
```

Comment:
`scan-credential` would be more precise.

Reply:
Remember to change nginx if you change this path

Reply:
I'm going to follow this pattern 
credential**Issued**QR: {
    layout: "fullWidthGrey",
    path: "/credentials/scan-**issued**/:credentialID",
  },
  credential**Link**QR: {
    layout: "fullWidthGrey",
    path: "/credentials/scan-**link**/:linkID",
  },

Reply:
I initially thought the same regarding `scan-credential` but later realized that the current URL is under the path `/credentials/` so I think it's ok as it's.

 * `/credentials/scan-link/{id}`
 * `/credentials/scan-issued/{id}`